articles/verifying-website-domains/index.md

```diff
@@ -0,0 +1,390 @@
+## Verifying Website Domains
+
+Verifying website domains for originality can be complicated during a time where the internet is constantly used for both good and bad. By using Python Shell and Windows PowerShell within a Windows environment, determining the credibility of a website domain name can be done.
+
+This topic could interest individuals and/or groups who would like to verify domain names for its validity and credibility through coding.
+
+At the end of this tutorial, it is possible to have a good grasp on retrieving DNS (Domain Name System) information by coding in Python Shell, skills developed or refreshed in website API commands within Windows PowerShell, and a tighter hold on identifying the complexities of domain modifications in an ever-changing technical space.
```

Comment:
This isn't very clear.

Website API commands?
To me, this implies you'll be interacting with a REST API which is not the case for this guide.

Reply:
I can clarify by shortening it.

I could just revise it to:
"At the end of this tutorial, it is possible to complete a complete a reverse DNS (Domain Name System) lookup using Python Shell and Windows PowerShell"

Would that make this portion clearer?